packages/devextreme/js/__internal/ui/menu/m_menu.ts

```diff
@@ -341,9 +349,11 @@ class Menu extends MenuBase {
 
     if (state) {
       this._hideVisibleSubmenu();
+      this._setRoleAria(state);
     } else {
       this._treeView && this._treeView.collapseAll();
       this._overlay && this._toggleTreeView(state);
+      this._setRoleAria(state);
```

Comment:
[copypaste]
So, u can move this operator out of `if/else` condition because u always call it

Reply:
[c317a55](https://github.com/DevExpress/DevExtreme/pull/28066/commits/c317a5579c7808781166a9bf40be333ba05ac031)